.github/workflows/docs-gen-and-push.yaml

```diff
@@ -102,23 +108,24 @@ jobs:
           w
           q
           EOF
-          mkdir -p content/en/${{ github.ref }}
-          mv content/en/main content/en/${{ github.ref }}
+          mv content/en/main content/en/${arr[2]}
           hugo --minify
-          rsync -av public/${{ github.ref }} ../../docs
+          rsync -av public/${arr[2]} ../../docs
 
+      # push older version HTML renders on gh-pages to be included in docs
       - name: Push to gh-pages on tag for archive
         if: startsWith(github.ref, 'refs/tags/v')
         run: |
           cd docs
-          git config --local user.name "github-actions[bot]"
-          git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config user.name "github-actions[bot]"
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
           git add -A
           git commit -m "Static archive of docs for ${{ github.ref }}"
-          git push
+          git push origin gh-pages
```

Comment:
This is pushing to the kcp repo's gh-pages branch? It's currently empty?

Reply:
Yes, this will archive the HTML render of the older docs versions. For now, it is empty.

Reply:
Also, the kcp docs site is not served from the gh-pages branch (as it used to do in past). GitHub has released an alternative mechanism to directly serve sites without utilizing a branch. We are using that.